docker-compose.dev.yml

```diff
@@ -0,0 +1,45 @@
+version: '3'
+
+services:
+  postgres:
+    image: postgres
+    restart: always
+    volumes:
+      - pgdata:/var/lib/postgresql/data
+    environment:
+      POSTGRES_USER: strapi
+      POSTGRES_PASSWORD: strapi
+      POSTGRES_DB: strapi
+    ports:
+      - '5432:5432'
+
+  mongo:
+    image: mongo
+    restart: always
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: root
+      MONGO_INITDB_ROOT_PASSWORD: strapi
+    volumes:
+      - mongodata:/data/db
+    ports:
+      - '27017:27017'
```

Comment:
Aren't you afraid that starting databases with their default ports would lead to conflict between local installs & docker ones?

Reply:
That's a good concern :) I didn't set different ports as I'm more worried about not changing the database.js file so people who are already using a docker config don't have to update there setup.

Reply:
You're right about backward compatibility.
Maybe we could add a little warning/info in the docs about the potential conflicts then?

Reply:
Sure Will add this to the readme ;)